numpy/core/src/umath/override.c

```diff
@@ -545,6 +521,27 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
                 break;
             }
         }
+        /*
+         * Set override arguments for each call since the tuple must
+         * not be mutated after use in PyPy
+         * We increase all references since SET_ITEM steals
+         * them and they will be DECREF'd when the tuple is deleted.
+         */
+        /* PyTuple_SET_ITEM steals reference */
```

Comment:
This line can now be removed, since you have the clearer description just above.